webapp/src/containers/Rhs/rhsData.tsx

```diff
@@ -103,11 +103,13 @@ const RhsData = ({
         );
     }, [getConfigState().data?.ServiceNowBaseURL]);
 
+    const errorTitle = (error && error === (Constants.GeneralErrorMessage + '\n' + Constants.ReconnectMessage)) ? error : '';
```

Comment:
I think we can remove this variable and can use the error directly below by using `includes`